content/backmatter.tex

```diff
@@ -501,6 +501,26 @@ \section{Version 1.4}
 \item Added the \FUNC{shmem\_calloc} function.
 \\ See Section \ref{subsec:shmem_calloc}.
 %
+\item Introduced the thread safe semantics that defines the interaction between \openshmem{} routines and User threads.
```

Comment:
We seem to use to both \openshmem{} and \openshmem. Do you want me to replace all places (in my ticket) or in the whole spec ?